kotishell/src/execute/execute_a.c

```diff
@@ -6,15 +6,29 @@
 /*   By: elenavoronin <elnvoronin@gmail.com>          +#+                     */
 /*                                                   +#+                      */
 /*   Created: 2023/07/25 15:48:11 by elenavoroni   #+#    #+#                 */
-/*   Updated: 2023/07/25 18:47:23 by dkolodze      ########   odam.nl         */
+/*   Updated: 2023/08/02 15:59:14 by dkolodze      ########   odam.nl         */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "execute.h"
 
-void	ex_execute(t_ks_kotistate *state, t_li_node *cmds)
+int	ex_execute(t_ks_kotistate *state, t_li_node *cmds)
 {
-	(void) state;
-	(void) cmds;
-	printf("I AM EXECUTING\n");
+	int					n_pipes;
+	t_ex_pipe_array		pipes;
+	int					retcode;
+
+	if (li_list_size(cmds) == 0)
```

Comment:
no command in bash does not reset status to 0 but retains the previous return code

Reply:
done